rslib/src/scheduler/fsrs/params.rs

```diff
@@ -636,4 +689,20 @@ pub(crate) mod tests {
             convert(revlogs, false)
         );
     }
+
+    #[test]
+    fn even_if_no_learning_steps_ignore_reviews_before_during_reviewing() {
```

Comment:
Just a naming nitpick:
```suggestion
    fn ignore_reviews_before_during_reviewing_even_if_no_learning_steps() {
```